src/composables/maxAmount.ts

```diff
@@ -19,23 +19,26 @@ import {
   validateTipUrl,
   checkAensName,
   handleUnknownError,
+  FUNGIBLE_TOKEN_CONTRACT_ACI,
 } from '../popup/utils';
 import { IAccount } from '../types';
 import { useGetter } from './vuex';
 import { useSdk } from './sdk';
+import { FungibleContractApi } from '../types/fungibleContract';
 
 export interface UseMaxAmountOptions {
   formModel: any,
 }
 
+let tokenInstance: Contract<FungibleContractApi>;
```

Comment:
Not a ref as `initializeContract` methods returns a reactive instance of the contract